system/doc/reference_manual/processes.xml

```diff
@@ -109,23 +185,503 @@ spawn(Module, Name, Args) -> pid()
+      <seeguide marker="expressions#send">send operator <c>!</c></seeguide>.
+      A received message can be fetched from the message queue by the
+      receiving process using the
+      <seeguide marker="expressions#receive"><c>receive</c></seeguide>
+      expression.
+    </p>
+
+    <marker id="sync-comm"/>
+    <p>
+      Synchronous communication can be broken down into multiple asynchronous
+      signals. An example of such a synchronous communication is a call to the
+      <seemfa marker="erts:erlang#process_info/2"><c>erlang:process_info/2</c></seemfa>
+      BIF when the first argument does not equal the process identifier of
+      the calling process. The caller sends an asynchronous signal requesting
+      information, and then blocks waiting for the reply signal containing
+      the requested information. When the request signal reaches its
+      destination, the destination process replies with the requested
+      information.
+    </p>
+
+    <section>
+      <title>Sending Signals</title>
+
+      <p>
+	Besides message signals there are a lot of other signals that can be
+	sent. Below follows a list of the most important signals. In all the
+	cases below of request/reply signal pairs, the request signal is sent
+	by the process calling the specific BIF, and the reply signal is sent
+	back to it when the requested operation has been performed.
+      </p>
+      <taglist>
+	<tag><c>message</c></tag>
+	<item>
+	  Sent when using the
+	  <seeguide marker="expressions#send">send operator <c>!</c></seeguide>,
+	  or when calling one of the
+	  <seemfa marker="erts:erlang#send/2"><c>erlang:send/2,3</c></seemfa>
+	  or
+	  <seemfa marker="erts:erlang#send_nosuspend/2"><c>erlang:send_nosuspend/2,3</c></seemfa>
+	  BIFs.
+	</item>
+
+	<tag><c>link</c></tag>
+	<item>
+	  Sent when calling the
+	  <seemfa marker="erts:erlang#link/1">link/1</seemfa> BIF.
+	</item>
+
+	<tag><c>unlink</c></tag>
+	<item>
+	  Sent when calling the
+	  <seemfa marker="erts:erlang#unlink/1">unlink/1</seemfa> BIF.
+	</item>
+
+	<tag><c>exit</c></tag>
+	<item>
+	  Sent either when explicitly sending an <c>exit</c> signal by
+	  calling the
+	  <seemfa marker="erts:erlang#exit/2">exit/2</seemfa> BIF, or when
+	  a <seeguide marker="#sending_exit_signals">linked process
+	  terminates</seeguide>. If the signal is sent due to a link, the
+	  signal is sent after all <seeguide marker="#visible-resources">
+	  <i>directly visible Erlang resources</i></seeguide> used by the
+	  process have been released.
+	</item>
+
+	<tag><c>monitor</c></tag>
+	<item>
+	  Sent when calling one of the
+	  <seemfa marker="erts:erlang#monitor/3">monitor/2,3</seemfa> BIFs.
+	</item>
+
+	<tag><c>demonitor</c></tag>
+	<item>
+	  Sent when calling one of the
+	  <seemfa marker="erts:erlang#demonitor/1">demonitor/1,2</seemfa>
+	  BIFs, or when a process monitoring another process terminates.
+	</item>
+
+	<tag><c>down</c></tag>
+	<item>
```

Comment:
Do we have any guarantee about the order of `down` signal vs `exit`?

Reply:
No